src/TeamData.elm

```diff
@@ -1,4 +1,4 @@
-module TeamData exposing (Model, Msg(..), Stations(..), init, nameCheck, stationToString, subscriptions, team, update, view)
+module TeamData exposing (Model, Msg(..), Stations(..), init, nameCheck, station, stationToString, subscriptions, team, update, view)
```

Comment:
You shouldn't expose `Msg`'s constructors. With exposed `Msg(..)` external code can construct a `TeamData` `Msg` and interfere with your state.